src/test/java/hudson/plugins/mercurial/MercurialSCMTest.java

```diff
@@ -72,4 +75,13 @@ public class MercurialSCMTest {
         assertEquals(EXPECTED_REPOSITORY_URL, actualEnvironment.get("MERCURIAL_REPOSITORY_URL"));
     }
 
+    @Issue("JENKINS-68562")
+    @Test public void abortIfSourceIsLocal() throws IOException {
+        try {
+            new MercurialSCM("", "https://mercurialserver/testrepo", "", "", "", null, true).abortIfSourceLocal();
+        } catch (AbortException e) {
+            fail("https source URLs should always be valid");
+        }
```

Comment:
or simply
```suggestion
        new MercurialSCM("", "https://mercurialserver/testrepo", "", "", "", null, true).abortIfSourceLocal();
```

Reply:
Yeah, I considered that as well, but I felt like having at least one assertion made the intent of the test a bit clearer.